platform-common/src/main/java/com/softicar/platform/common/ui/image/Images.java

```diff
@@ -129,4 +130,34 @@ public static long countPixelsWithColor(BufferedImage image, Color expectedColor
 
 		return matchingPixels;
 	}
+
+	/**
+	 * Determines whether {@link #readImages(Supplier)} can handle images with
+	 * the given {@link IMimeType}.
+	 *
+	 * @param mimeType
+	 *            the image {@link IMimeType} (never <i>null</i>)
+	 * @return <i>true</i> if the given image {@link IMimeType} is supported;
+	 *         <i>false</i> otherwise
+	 */
+	public static boolean isSupported(IMimeType mimeType) {
```

Comment:
since this only tests for image readers, maybe the name should be `isReadable` or `canRead` or `hasReader`, you get the point :)